jsman.py

```diff
@@ -19,11 +19,8 @@ def getFirstResult(searchResult):
 
 
 def openUrl(url):
-    if platform.system() == 'Darwin':
-        os.system("open " + url)
-    else:
-        os.system("firefox " + url)
-
+    open_cmd = 'open' if platform.system() == 'Darwin' else 'gnome-open'
```

Comment:
Thanks for the PR. 
Could you keep `firefox` as the default? Promoting firefox is one of my core value. We can use some kind of configuration to override the default (config file, environment variable or command line argument)
Also, please avoid inline `if...else`, because we many need to support more than two `platform.system()` cases in the near future.